pyfixest/did/saturated_twfe.py

```diff
@@ -151,23 +158,31 @@ def test_treatment_heterogeneity(self) -> pd.Series:
         )
 
     def aggregate(
-        self, agg="period", weighting: Optional[str] = "shares"
+        self,
+        agg="period",
+        use_weights: bool = True,
```

Comment:
Asking the same question three times, sorry for that (😅 )- would it not make more sense to assume that if weights have been provided in the model fit, they should also be applied in the aggregation?